ui/raidboss/data/05-shb/raid/e2n.js

```diff
@@ -301,17 +229,7 @@
         return data.spell[matches.target] == 'eye';
       },
       delaySeconds: 2,
-      alarmText: function(data, matches) {
-        if (data.me != matches.target) {
-          return {
-            en: 'Look Away from ' + data.ShortName(matches.target),
-            de: 'Von ' + data.ShortName(matches.target) + ' weg schauen',
-            fr: 'Ne regardez pas ' + data.ShortName(matches.target),
-            cn: '背对 ' + data.ShortName(matches.target),
-            ko: '보지마세요 -> ' + data.ShortName(matches.target),
-          };
-        }
-      },
+      response: Responses.lookAwayFrom(),
```

Comment:
/home/travis/build/quisquous/cactbot/ui/raidboss/data/05-shb/raid/e2n.js: E2N Countdown Marker Shadoweye cannot have both 'response' and 'infoText'

You can't have infoText here too